packages/frontend/src/components/NavBar/MenuList.tsx

```diff
@@ -18,30 +20,84 @@ export const MenuList = () => {
       toggleDrawer();
     }
   };
+  const { pathname } = useLocation();
+  const routeName = getRouterName(pathname).split('/').filter(Boolean)[0];
 
   return (
     <>
       {MenuItems.map((item: IMenuItem) => {
         const { key, primary, icon, to, children } = item;
+
+        const selected =
+          (key !== 'Logout' && capitalize(getRouterName(primary as string)) === capitalize(routeName)) ||
+          (key === 'Landing' && pathname === '/');
         return (
           <Fragment key={key}>
             <ListItem disablePadding alignItems="flex-start" onClick={() => categoryHandler(key)}>
-              <ListItemButton href={to ? to : ''}>
+              <ListItemButton
+                href={to ? to : ''}
+                selected={selected}
+                sx={{
+                  '&.Mui-selected': {
+                    backgroundColor: '#fff',
+                    '&::before': {
+                      content: '""',
+                      position: 'absolute',
+                      left: 0,
+                      top: 0,
+                      bottom: 0,
+                      width: '4px',
+                      backgroundColor: Colors.midblue,
+                    },
+                    '&.MuiListItemButton-root:hover': {
+                      backgroundColor: theme.palette.action.hover,
+                    },
+                  },
+                }}
+              >
                 <ListItemIcon>{icon}</ListItemIcon>
-                <ListItemText primary={primary} sx={{ opacity: openDrawer ? 1 : 0 }} />
+                <ListItemText
+                  primary={primary}
+                  sx={{
+                    opacity: openDrawer ? 1 : 0,
+                    '.MuiListItemText-primary': {
+                      fontWeight: selected ? 'bold' : 'normal',
```

Comment:
Same as commented below, you could try using `fontFamily: 'Exo2-Bold' or 'Exo2-ExtraBold'` for bold text